content/pages/sprint/2022/04-taipei.rst

```diff
@@ -0,0 +1,77 @@
+==================================
+Scisprint 2022 April in Taipei
+==================================
+
+:date: 2022-03-17 20:00
+:url: sprint/2022/04-taipei
+:save_as: sprint/2022/04-taipei.html
+
+Scisprint 2022 April in Taipei
+==================================
+
+We cordially invite you to join the Scisprint 2022 April in Taipei (at a
+`cozy meeting room <#venue>`__).  We will be using 4 hours to discuss and code for the
+following projects:
+
+* Python 官方說明文件臺灣繁體中文翻譯計畫: https://github.com/python/python-docs-zh-tw
+* modmesh: https://github.com/solvcon/modmesh
+
+If you want to join and bring your projects, please `Contact us`_.
+
+Meeting Minutes and Registration
+--------------------------------
+
+Add your name to the sprint note: https://hackmd.io/@sciwork/HJ_gzk-Gq .
+
+Date & time
+-----------
+
+9th April, Saturday, 2022, 14:00 -- 18:00
+
+Fee
```

Comment:
Fee is OK.  Alternate wording is cost sharing.

Reply:
Cool, seems straightforward.